code/modules/wiremod/shell/airlock.dm

```diff
@@ -110,6 +121,12 @@
 	is_open.set_output(FALSE)
 	closed.set_output(COMPONENT_SIGNAL)
 
+/obj/item/circuit_component/airlock/proc/on_airlock_touched(datum/source, user)
+	SIGNAL_HANDLER
+	. = COMPONENT_PREVENT_OPEN
```

Comment:
okay but it always prevent airlock from opening?

Reply:
There's no way to open the airlock without using the circuit anyway so it doesn't really change anything